htdocs/compta/facture/card-rec.php

```diff
@@ -356,6 +362,11 @@
 		// Set bank account
 		$object->context['actionmsg'] = $langs->trans("FieldXModified", $langs->transnoentitiesnoconv("Bank"));
 		$result = $object->setBankAccount(GETPOSTINT('fk_account'));
+	} elseif ($action == 'setbankaccountcustomer' && $usercancreate) {
+		// Set bank account customer
+		$object->context['actionmsg'] = $langs->trans("FieldXModified", $langs->transnoentitiesnoconv("BankCustomer"));
```

Comment:
you need to add missing translation `BankCustomer` in `langs/en_US/banks.lang` only

Reply:
It's corrected, thank you !